client/cl_boss.lua

```diff
@@ -154,6 +156,17 @@ RegisterNetEvent('qb-bossmenu:client:ManageEmployee', function(data)
             }
         }
     end
+
+    EmployeeMenu[#EmployeeMenu + 1] = {
+        header = Lang:t('body.dutyemp'),
+        icon = not not data.player.onduty and 'fa-solid fa-user-minus' or 'fa-solid fa-user-plus',
```

Comment:
`PlayerData.job.onduty` is checked at each player login for a nil value, if `onduty` is nil it is set to the default duty value which makes `not not` unnecessary though if `onduty` wasn't checked, this would be a good workaround.